ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountStatsMySqlStore.java

```diff
@@ -166,14 +164,23 @@ public AccountStatsMySqlStore(AccountStatsMySqlConfig config, DataSource dataSou
     partitionClassReportsDao = new PartitionClassReportsDao(dataSource, mySqlMetrics);
     this.hostnameHelper = hostnameHelper;
     storeMetrics = new AccountStatsMySqlStore.Metrics(registry);
-    if (!Strings.isNullOrEmpty(localBackupFilePath)) {
-      // load backup file and this backup is the previous stats
-      ObjectMapper objectMapper = new ObjectMapper();
+    if (config.readPreviousStatsFromMysqlDb) {
+      long startTimeMs = System.currentTimeMillis();
       try {
-        this.previousStats = objectMapper.readValue(new File(localBackupFilePath), StatsWrapper.class);
+        this.previousStats = queryAccountStatsBySimplifiedHostName(hostname, startTimeMs);
       } catch (Exception e) {
         this.previousStats = null;
```

Comment:
if the load from mysql fails, do we want to fall back to reading from the local backup file?

Reply:
Sure, I think we can do that. Updated.